src/pages/admin/concerns.tsx

```diff
@@ -0,0 +1,38 @@
+import { useRouter } from 'next/router'
+
+import ConcernsCard from '@/components/Admin/concernscard'
+import NavBar from '@/components/NavBar'
+import Button from '@/components/UI/button'
+import { useVetConcerns } from '@/hooks/vetconcerns'
+import { VetConcern } from '@/types/types'
+
+const Output = () => {
+  const { data: concerns } = useVetConcerns()
+  const router = useRouter()
+
+  return (
+    <>
+      <div className='m-auto flex h-14 w-full flex-row'>
+        <div className='m-auto flex-1 text-center'>
+          <Button
+            type='button'
+            size='medium'
+            onClick={() => router.push('/admin')}
+          >
+            Back
+          </Button>
+        </div>
+        <h1 className='m-3 flex-1 text-center text-2xl'>Concerns</h1>
+
+        <div className='flex-1'></div>
```

Comment:
Is this necessary? Doesn't seem like its doing anything.

Reply:
Ah I see why, but this is bad practice and just indicates that there is a larger CSS issue at hand. 

You could simplify the code by giving the button a fixed position and centering the Concerns text.

Something like this:

```
<div className='mx-2 mt-2'>
          <Button
            type='button'
            size='medium'
            onClick={() => router.push('/admin')}
            className='fixed top-2 left-2'
          >
            Back
          </Button>

          <h1 className='text-center text-2xl'>Concerns</h1>

          <div className='pt-2'>
            {concerns?.map((concern: VetConcern, i: number) => (
              <ConcernsCard key={i} {...concern}></ConcernsCard>
            ))}
          </div>
  <NavBar />
</div>
```